terraform/common/modules/domains/data.tf

```diff
@@ -0,0 +1,9 @@
+data aws_route53_zone selected {
+  # AWS treats *.domain.name and domain.name as the same validation record
```

Comment:
Is this comment still useful?
Also, `salected` was approriate when we were filtering. Now we could have a more generic name like `all` or `zones`

Reply:
Removed comment
Changed from `selected` to `zone`